<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>JoyMouse</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QComboBox" name="joypadComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Choose a Gamepad to configure</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="activatedCheckBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Activate Selected Gamepad</string>
        </property>
        <property name="text">
         <string>Activated</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QComboBox" name="profileComboBox"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QGroupBox" name="buttonGroupBox">
        <property name="toolTip">
         <string>Button mappings</string>
        </property>
        <property name="title">
         <string>Buttons</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2"/>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="gamepadLabel">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="resources.qrc">:/Assets/Gamepad_configure.jpg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="axisGroupBox">
        <property name="toolTip">
         <string>Axis mappings</string>
        </property>
        <property name="title">
         <string>Axis</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3"/>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="mapButton">
        <property name="toolTip">
         <string>Map a Button/Axis</string>
        </property>
        <property name="text">
         <string>Map Button</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="mapAxisButton">
        <property name="text">
         <string>Map Axis</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sensivityButton">
        <property name="text">
         <string>Sensivity</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="saveProfileButton">
        <property name="toolTip">
         <string>Save the Profile</string>
        </property>
        <property name="text">
         <string>Save Profile</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_Profile"/>
    <addaction name="actionSave_Profile"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoad_Profile">
   <property name="text">
    <string>Load Profile</string>
   </property>
  </action>
  <action name="actionSave_Profile">
   <property name="text">
    <string>Save Profile</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
